lib/smartdown_flows/student-finance-forms-v2/student-finance-forms-v2.txt

```diff
@@ -0,0 +1,29 @@
+meta_description: Download Student Finance England application forms and guidance notes - including forms PN1, PR1, PTG1, PFF2 and CYI
+satisfies_need: 100982
+status: published
+
+# Student finance forms
+
+Download student finance forms and guidance notes for Student Finance England including:
+
++ forms PN1, PR1, PTL1, PFF2 and CYI
++ forms for parents
++ forms for EU students
+
+[start: type_of_student?]
+
+##Before you start
+
+The forms are different for students from [Scotland](http://www.saas.gov.uk), [Wales](http://www.studentfinancewales.co.uk) and [Northern Ireland](http://www.studentfinanceni.co.uk).
+
+For braille or alternative formats contact the helpline. If you email, include your contact details and the format you need.
+
+
+$C
+**Helpline - alternative formats only**
+
+0141 243 3686<br />
+<brailleandlargefonts@slc.co.uk>
+$C
+
+* otherwise => type_of_student
```

Comment:
You need an otherwise to tell the coversheet which of the questions is the first one :)